web_widget_mail_send_odoo/static/src/css/web_widget_mail_send_odoo.css

```diff
@@ -0,0 +1 @@
+.openerp .oe_form .oe_form_field_email_intern input,
```

Comment:
people who don't understand German will be a bit puzzled by *intern*, consider replacing this by *internal* at all occurences